### ISSUE TITLE
Light Documentation 

### ISSUE THREAD

**Issue by srothst1** [CesiumGS/cesium#9756]
Our documentation for the `Light` object is not very comprehensive. While the member `color` has decent documentation, information on the member `intensity` is scarce. `intensity` is a number, but it is unclear if increasing the number increases the intensity. It is also unclear what the maximum `intensity` value can be. 

If someone helps me get up to speed on `Light`, I will promptly submit a pull request addressing the shortcomings of our documentation. 

`Light` documentation: https://cesium.com/learn/cesiumjs/ref-doc/Light.html

Issue stems from the following community forum posts: https://community.cesium.com/t/image-only-on-the-top-side-of-poligon/14863

@lilleyse 
@ebogo1

**Comment by lilleyse** [comment]
Yeah this could definitely be documented better. There's a more detailed description in the original PR: https://github.com/CesiumGS/cesium/pull/8493

The intensity controls the strength of the light. The minimum value is 0.0. No maximum value. Together the color and intensity produce a high-dynamic-range light color. `intensity` can also be used as a convenience to just dim or brighten the light without changing the hue.

It's similar to

* https://threejs.org/docs/#api/en/lights/PointLight
* https://doc.babylonjs.com/typedoc/classes/babylon.light
